notify.go

```diff
@@ -83,14 +83,14 @@ func notify(config *effectiveConfig, hook hookEvent, ws string, notification gri
 		context := buildContext(hook, ws, logDir)
 		message, color, err := notification.HipchatNotification(context, config)
 		if err != nil {
-			return err
+			return message, err
 		}
 
 		err = sendMessageToRoom(config.hipChatToken, config.hipChatRoom, config.grimServerID, message, color)
 		if err != nil {
-			return err
+			return message, err
 		}
 	}
 
-	return ghErr
+	return "", ghErr
```

Comment:
this logic will cause messages to be logged only if hipchat is configured.  and no message to be logged when hipchat is not configured. 

That was undefined in the issue I believe, but my personal opinion is that better behavior would be to log teh message no matter what and only send to hipchat if it is configured.